CONTRIBUTING.md

```diff
@@ -58,12 +58,12 @@ docker-compose up --build
 
 The command above will create a container with all the required dependencies and start a mock server at `localhost:8080`.
 
-#### Local `@types/*`
+#### Local `types definition`
```

Comment:
To be honest, I would remove the ticks around _types definition_, I think they don't apply anymore.